php/composer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "varias/php",
+    "type": "project",
+    "description": "Varias PHP standards",
+    "require": {
+        "php": "^8.1"
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "scripts": {
+        "phpcs-fix": "tools/php-cs-fixer/changes-only",
+        "phpcs": "tools/php-cs-fixer/changes-only -v --dry-run",
```

Comment:
**suggestion:** I would opt to remove this `changes-only` and just run phpcs on all files. Or maybe add the changes-only as an option if someone wants to run it a bit faster locally. In CI (github actions), we can always just run the whole project, because we want the whole project to be valid. This will also catch the use-case where we change the phpcs config to add or change a rule, so you'd have to fix the files in that same PR (instead of those files not being caught)

Reply:
This can be applied on a project per project basis depending on the needs of the devs